### PR TITLE
Add crs and resolution to EDC BYOC collections

### PR DESCRIPTION
**Pull request by zansinergise**
https://git.sinergise.com/team-6/openeo-platform/-/issues/109

### CODE REVIEW

File: collections/corine-land-cover-accounting-layers.yaml [hunk 1]
```diff
@@ -98,5 +100,15 @@ CubeDimensions:
     values:
       - CLC_ACC
       - dataMask
+Summaries:
+  raster:bands:
+    - name: CLC_ACC
+      openeo:gsd:
```
Comment (dthiex) [inline collections/corine-land-cover-accounting-layers.yaml:106]: Please also add everywhere a description of the band. The info should always (in most cases) be somewhere in the yaml file (for the CLC_ACC band it's in line 27 so `description: "Corine Land Cover Accounting Layer"`.

File: collections/corine-land-cover-accounting-layers.yaml [hunk 2]
```diff
@@ -98,5 +100,15 @@ CubeDimensions:
...
+        value:
+          - 100
+          - 100
+        unit: m
+    - description: The mask of data/no data pixels.
+      name: dataMask
 RegistryEntryAdded: "2021-10-20"
 RegistryEntryLastModified: "2022-02-10"
```
Comment (dthiex) [inline collections/corine-land-cover-accounting-layers.yaml:114]: Nitpick: please update the date here.

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/corine-land-cover-accounting-layers.yaml (hunk 1 above)
Comment (zansinergise) [reply collections/corine-land-cover-accounting-layers.yaml:107]: Added descriptions of bands

File: collections/corine-land-cover-accounting-layers.yaml (hunk 2 above)
Comment (zansinergise) [reply collections/corine-land-cover-accounting-layers.yaml]: Fixed

File: collections/seasonal-trajectories.yaml [hunk 3]
```diff
@@ -123,5 +187,172 @@ CubeDimensions:
...
+        unit: m
+    - description: Quality Flag
+      name: QFLAG
+      openeo:gsd:
+        value:
+          - 10
+          - 10
+        unit: m
+    - description: The mask of data/no data pixels.
+      name: dataMask
```
Comment (dthiex) [inline collections/seasonal-trajectories.yaml:212]: Same here not sure if there is a dataMask band for this collection.

**Review by dthiex — CHANGES_REQUESTED** [review]

**Review by dthiex — APPROVED** [review]
Looks good now.

**Pull request merged**